config/crd/bases/compliance.openshift.io_tailoredprofiles.yaml

```diff
@@ -57,13 +57,18 @@ spec:
                   description: RuleReferenceSpec specifies a rule to be selected/deselected,
                     as well as the reason why
                   properties:
+                    kind:
+                      description: Type of rule, either "Rule" or "CustomRule". "Rule"
+                        is the default if not specified.
+                      type: string
                     name:
                       description: Name of the rule that's being referenced
                       type: string
                     rationale:
                       description: Rationale of why this rule is being selected/deselected
                       type: string
                   required:
+                  - kind
```

Comment:
Does this mean it's possible to have a `CustomRule` named `foo` and a `Rule` named `foo`?

I'm trying to think through a case where someone would want to disable a `CustomRule`. In that case, wouldn't they just not include it in the tailored profile?

Reply:
> In that case, wouldn't they just not include it in the tailored profile?
thanks right,